src/platform/tigerlake/include/platform/lib/ams.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ */
+
+#ifdef __SOF_LIB_AMS_H__
+
+#ifndef __PLATFORM_LIB_AMS_H__
+#define __PLATFORM_LIB_AMS_H__
+
+/* max number of message UUIDs */
+#define AMS_SERVICE_UUID_TABLE_SIZE 16
+/* max number of async message routes */
+#define AMS_ROUTING_TABLE_SIZE 16
+/* Space allocated for async message */
+#define AMS_MAX_MSG_SIZE 0x1000
```

Comment:
the values look exactly the same as for MTL, which begs the question why those values need to be platform-specific?